WebRoot/css/zm.css

```diff
@@ -7679,3 +7669,91 @@ input[type="radio"] + label {
 .ZmAppointmentTooltip TD.ZmAppointmentTooltipValue {
 	@AppointmentTooltipValue@	
 }
+
+/* Appointment create/edit view */
+.ZmApptEditView {
+	@ApptEditView@
+}
+
+.ZmApptEditView * {
+	@ApptEditViewFontSize@
+}
+
+.ZmApptEditView .ZmSchedulerGridHeaderCell {
+	@FontSize-smaller@
```

Comment:
@ApptEditSchedulerHeader@

Reply:
This rule is already declared at https://github.com/Zimbra/zm-web-client/blob/9d04f275f05446628a40003f582939ebf6e142e2/WebRoot/css/zm.css#L3301, here are we just preserving it's original fontSize declared at https://github.com/Zimbra/zm-web-client/blob/9d04f275f05446628a40003f582939ebf6e142e2/WebRoot/css/zm.css#L3304
Because it's getting overriden here https://github.com/Zimbra/zm-web-client/pull/143#discussion-diff-138814907R7679